test/unix/common.mli

```diff
@@ -12,6 +12,13 @@ module Key : sig
   val v : unit -> t
 end
 
+(** Simple key/value modules with String type and a random constructor *)
+module KeyDS : sig
```

Comment:
I think I'd be in favor of replacing the simple string module for values, and use this one everywhere instead.

Reply:
I'm not really sure to understand what you're saying here :-)

Reply:
Done in the last update